Assets/Scripts/SimulationManager.cs

```diff
@@ -91,41 +105,26 @@ public float CurrentScaleFactor(float sceneRadius)
     // Movement synchronization throttled for Heroku/Mongo
     public float MovementSendInterval = 1.0f;
     
-    // Server Address!
-    public string LocalNetworkServer = "ws://localhost:2567";
-    public string DevNetworkServer = "ws://ceasar-serve-170349161-ceutdkz.herokuapp.com/";
-    public string ProductionNetworkServer = "ws://ceasar-server-staging.herokuapp.com/";
-
     public StarComponent CurrentlySelectedStar;
-    private DateTime _currentSimulationTime = DateTime.UtcNow;
 
     public DateTime CurrentSimulationTime
     {
-        get { return _currentSimulationTime; }
-        set
-        {
-            _currentSimulationTime = value;
-        }
-    }
-    private bool useCustomSimTime = false;
-    public bool UseCustomSimulationTime
-    {
-        get { return useCustomSimTime; }
+        get { return LocalPlayer.Pin.SelectedDateTime; }
         set
         {
-            useCustomSimTime = value;
-            if (!useCustomSimTime) CurrentSimulationTime = DateTime.UtcNow;
+            LocalPlayer.Pin.SelectedDateTime = value;
         }
     }
 
-    private LatLng currentLocation = new LatLng {Latitude = 0, Longitude = 0};
+    private LatLng _currentLatLng = new LatLng {Latitude = 0, Longitude = 0};
 
-    public LatLng Currentlocation
+    public LatLng CurrentLatLng
     {
-        get { return currentLocation; }
+        get { return _currentLatLng; }
         set
         {
-            currentLocation = value;
+            _currentLatLng = value;
```

Comment:
Just noticing that we are saving this value in two places.  Can they diverge?

Reply:
I wasn't sure I felt comfortable with deleting them but reviewing where they were used, it makes little sense to duplicate. Will fix, thanks!